polars/lazy/functions.ts

```diff
@@ -630,9 +630,90 @@ export function struct(
   *  │ 3   ┆ 2   ┆ [6, 4]      │
   *  └─────┴─────┴─────────────┘
  */
+
 export function element(): Expr {
   return col("");
 }
+
+/** Compute the bitwise AND horizontally across columns.
+
+    Parameters
+    ----------
+    *exprs
+        Column(s) to use in the aggregation. Accepts expression input. Strings are
+        parsed as column names, other non-expression inputs are parsed as literals.
+
+    Examples
+    --------
+    >>> const df = pl.DataFrame(
+    ...     {
+    ...         "a": [False, False, True, True],
+    ...         "b": [False, True, None, True],
+    ...         "c": ["w", "x", "y", "z"],
+    ...     }
+    ... )
+    >>> df.with_columns(pl.all_horizontal("a", "b"))
```

Comment:
looks like you missed updating some variables to JS syntax

Reply:
Yep, used them for examples and then forgot that JS and python are in fact two different languages when it comes to syntax on true/false/null, fixed now